torch/csrc/jit/ir/ir.h

```diff
@@ -435,6 +435,11 @@ struct TORCH_API Node {
   bool isNondeterministic() const;
   bool hasSideEffects() const;
 
+  // instructions lowered by the interpreter and not run in the optimized graph
+  bool notExecutedOp() const {
+    return kind_ == prim::Constant || kind_ == prim::profile;
```

Comment:
nitpick: do we add `prim::profile_optional` later on?

Reply:
yea, we do (although maybe we hadnt at the point of your reviewing)